### DISCUSSION TITLE
Syncing Courseplay on two PC's

### DISCUSSION THREAD

**Discussion by Torqueitup** [Courseplay/Courseplay_FS22#3064] [Q&A] [answered]
I have two PC's on which I play FS22 and would like to be able to Sync the Courseplay data across the two machines.
What CP files do I need to sync and what would be their typical locations in a standard directory setup?
Any help is much appreciated.

**Comment by Tensuko** [comment, marked as answer]
modSettings folder
There is everything from course, custom fields and user settings.
Thats the base folder for mods where they should write all settings that don't belong in the savegame.

**Reply by Torqueitup** [reply]
Thanks very much for the help.  I'll get my SyncbackPro to look after things now.
